requirements-extra.txt

```diff
@@ -12,3 +12,7 @@ mink
 
 # for extra robosuite_models
 robosuite-models==1.0.0
+
+# for USD export and rendering
+usd-core
+tqdm
```

Comment:
can we just make tqdm part of the main requirements in setup.py / requirements.py? it's a very common utility